readme.markdown

```diff
@@ -80,6 +80,12 @@ options are:
     * getNodeModulesDirs - a thunk (no-argument function) that returns the paths using standard `node_modules` resolution
     * opts - the resolution options
 
+* `opts.packageIterator(request, start, opts)` - return the list of candidate paths where the packages sources may be found (probably don't use this)
+    * request - the import specifier being resolved
+    * start - lookup path
+    * getPackageCandidates - a thunk (no-argument function) that returns the paths using standard `node_modules` resolution
```

Comment:
this really does confuse me. a user provides `packageIterator` and their callback receives 4 arguments - why would they receive a thunk here rather than passing through the request and the start?

Reply:
My intent was to use the same pattern as `paths`, which takes a thunk here:

https://github.com/browserify/resolve/blob/master/lib/node-modules-paths.js#L35

Reply:
hmm, fair point; i made that change in https://github.com/browserify/resolve/pull/173#pullrequestreview-177499139 on top of #172. I suppose we'll stick with it.